firebase-ml-modeldownloader/firebase-ml-modeldownloader.gradle

```diff
@@ -43,25 +60,43 @@ android {
 }
 
 dependencies {
+    implementation project(':encoders:firebase-encoders')
+    implementation project(':encoders:firebase-encoders-json')
     implementation project(':firebase-common')
     implementation project(':firebase-components')
+    implementation project(':firebase-datatransport')
     implementation project(':firebase-installations-interop')
+    implementation project(':transport:transport-api')
     runtimeOnly project(':firebase-installations')
 
+    implementation 'androidx.annotation:annotation:1.1.0'
     implementation 'com.google.android.gms:play-services-tasks:17.2.0'
     implementation 'com.google.auto.service:auto-service-annotations:1.0-rc6'
     implementation 'javax.inject:javax.inject:1'
+    compile 'com.google.protobuf:protobuf-lite:3.0.0'
```

Comment:
this is the wrong protolite runtime, you need to use protobuf-javalite(see. Also note that you should add it under `testImplementation` not `compile` or `implementation`, as doing so will make this a dependency of your SDK, not just your tests.

See example here https://github.com/firebase/firebase-android-sdk/blob/f4d1987fdad3932f7fa5a9fb7c61a24b725e6f62/firebase-crashlytics-ndk/firebase-crashlytics-ndk.gradle#L128